gatsby-site/src/pages/index.js

```diff
@@ -1,13 +1,15 @@
 import React from 'react'
 import Link from 'gatsby-link'
+import Team from '../components/team/team'
 
 const IndexPage = () => (
   <div>
     <h1>Hi people</h1>
     <p>Welcome to your new Gatsby site.</p>
     <p>Now go build something great.</p>
-    <p>Hello, I'm new.</p>
+    <p>Hello, I&#39;m new.</p>
     <Link to="/page-2/">Go to page 2</Link>
+    <Team/>
```

Comment:
We will probably need a coordinated way to merge this with @lianthompson's changes in the other PR.